app/lib/scorecard/backend.dart

```diff
@@ -46,7 +46,9 @@ class ScoreCardBackend {
     String packageName,
     String packageVersion, {
     @required bool onlyCurrent,
+    List<String> requiredReportTypes,
```

Comment:
Do we ever call this with anything ,but `null`?

How about we already reload unless both types are present? It's simpler and we have fewer options to deal with...
The overhead will likely be small?

Reply:
If adding new options, these should be documented... :)